### PR TITLE
Fix references

### PR DESCRIPTION
**Pull request by MorrisJobke**

### CODE REVIEW

**Comment by schiessle** [comment]
:+1:

**Comment by LukasReschke** [comment]
LGTM

File: apps/encryption/l10n/sq.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "New recovery key password" : "Fjalëkalimi i ri kyçi rimarrjesh",
     "Repeat new recovery key password" : "Rijepni fjalëkalimin e ri kyçi rimarrjesh",
     "Change Password" : "Ndryshoni Fjalëkalimin",
-    "ownCloud basic encryption module" : "modul i thjeshtë ownCloud fshehtëzimesh",
+    "basic encryption module" : "modul i thjeshtëfshehtëzimesh",
```
Comment (vincchan) [inline apps/encryption/l10n/sq.json]: @MorrisJobke don't think this is supposed to be written together

File: apps/encryption/l10n/fr.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "New recovery key password" : "Nouveau mot de passe de la clé de récupération",
     "Repeat new recovery key password" : "Répétez le nouveau mot de passe de la clé de récupération",
     "Change Password" : "Changer de mot de passe",
-    "ownCloud basic encryption module" : "Module de chiffrement de base d'ownCloud",
+    "basic encryption module" : "Module de chiffrement de base d'",
```
Comment (vincchan) [inline apps/encryption/l10n/fr.json]: @MorrisJobke remove " d' "

File: apps/encryption/l10n/gl.json [hunk 3]
```diff
@@ -37,7 +37,7 @@
     "New recovery key password" : "Novo contrasinal da chave de recuperación",
     "Repeat new recovery key password" : "Repita o novo contrasinal da chave de recuperación",
     "Change Password" : "Cambiar o contrasinal",
-    "ownCloud basic encryption module" : "Módulo básico de cifrado de ownCloud",
+    "basic encryption module" : "Módulo básico de cifrado de",
```
Comment (vincchan) [inline apps/encryption/l10n/gl.json]: @MorrisJobke remove "de"

File: apps/encryption/l10n/it.json [hunk 4]
```diff
@@ -44,7 +44,7 @@
     "New recovery key password" : "Nuova password della chiave di ripristino",
     "Repeat new recovery key password" : "Ripeti la nuova password della chiave di ripristino",
     "Change Password" : "Modifica password",
-    "ownCloud basic encryption module" : "Modulo di cifratura base di ownCloud",
+    "basic encryption module" : "Modulo di cifratura base di",
```
Comment (vincchan) [inline apps/encryption/l10n/it.json]: @MorrisJobke remove "di"

File: apps/encryption/l10n/oc.json [hunk 5]
```diff
@@ -42,7 +42,7 @@
     "New recovery key password" : "Novèl senhal de la clau de recuperacion",
     "Repeat new recovery key password" : "Repetissètz lo novèl senhal de la clau de recuperacion",
     "Change Password" : "Cambiar de senhal",
-    "ownCloud basic encryption module" : "Modul de chiframent de basa d'ownCloud",
+    "basic encryption module" : "Modul de chiframent de basa d'",
```
Comment (vincchan) [inline apps/encryption/l10n/oc.json]: @MorrisJobke remove "d'"

File: apps/encryption/l10n/pt_PT.json [hunk 6]
```diff
@@ -44,7 +44,7 @@
     "New recovery key password" : "Nova palavra-passe da chave de recuperação",
     "Repeat new recovery key password" : "Repetir palavra-passe da chave de recuperação",
     "Change Password" : "Alterar Palavra-passe",
-    "ownCloud basic encryption module" : "módulo de encriptação básico da ownCloud",
+    "basic encryption module" : "módulo de encriptação básico da",
```
Comment (vincchan) [inline apps/encryption/l10n/pt_PT.json]: @MorrisJobke remove "da"

File: apps/encryption/l10n/sl.js [hunk 7]
```diff
@@ -44,7 +44,7 @@ OC.L10N.register(
     "New recovery key password" : "Novi ključ za obnovitev gesla",
     "Repeat new recovery key password" : "Ponovi novi ključ za obnovitev gesla",
     "Change Password" : "Spremeni geslo",
-    "ownCloud basic encryption module" : "Osnovno šifriranje z ownCloud",
+    "basic encryption module" : "Osnovno šifriranje z",
```
Comment (vincchan) [inline apps/encryption/l10n/sl.js]: @MorrisJobke remove "z"

File: apps/encryption/l10n/sl.json [hunk 8]
```diff
@@ -42,7 +42,7 @@
     "New recovery key password" : "Novi ključ za obnovitev gesla",
     "Repeat new recovery key password" : "Ponovi novi ključ za obnovitev gesla",
     "Change Password" : "Spremeni geslo",
-    "ownCloud basic encryption module" : "Osnovno šifriranje z ownCloud",
+    "basic encryption module" : "Osnovno šifriranje z",
```
Comment (vincchan) [inline apps/encryption/l10n/sl.json]: see above

**Comment by vincchan** [comment]
@MorrisJobke whoops!! I was just reviewing the files when you merge the PR :/

**Comment by MorrisJobke** [comment]
@vincchan I know that there are issues in some languages. Those then needs to be fixed in the transifex setup.

**Comment by MorrisJobke** [comment]
I just talked with @schiessle and it's maybe better to do on stable9 a s/ownCloud/Nextcloud/ and on master a proper fixing of the strings. That then keeps the translations for the next releases.

**Pull request merged**